### ISSUE TITLE
URL Shortener

### ISSUE THREAD

**Issue by SnakeO** [FileShuttle/fileshuttle#19]
sht.tl is online and working. I think it's a better vanity URL, especially for this app.

**Comment by capi1O** [comment]
It is down right now. Better not rely on an external service IMHO.

**Comment by SnakeO** [comment]
You sure? It's up for me..

**Comment by capi1O** [comment]
Yes it it up now, it was definitely down yesterday (it caused some problems at work).
I unchecked the option (shorten URL) in case it happens again.
